### PR TITLE
Fix wordings

### LINKED ISSUE #30719: Release 8.1.0 🚀
This issue tracks the status of the PrestaShop 8.1.0 release

### Information

Q                                                     | A
--------------------------------|---------------
**Type**                                        | Minor version
**Milestone**                                | [8.1.0](https://github.com/PrestaShop/PrestaShop/milestone/84)
**Kanban**                                    | [8.1.x](https://github.com/PrestaShop/PrestaShop/projects/28)
**[Feature Freeze](https://build.prestashop-project.org/news/2023/prestashop-8-1-feature-freeze/)** | 2023-01-31
**[Beta](https://build.prestashop-project.org/news/2023/prestashop-8-1-beta-release/)** | 2023-02-17
**[Release candidate 1](https://build.prestashop-project.org/news/2023/prestashop-8-1-rc1-release/)**             | 2023-05-25
**Est. Final release**   | 2023-06-26
**Release manager tech**           | @lartist @matks 
**Release manager product**     |   @fabienvallon 
**Release manager QA**              |  @MhiriFaten 

### Current status
- [x] [Specification](https://github.com/PrestaShop/PrestaShop/issues/30292)
- [x] Feature development
- [x] [Feature freeze](https://build.prestashop-project.org/news/2023/prestashop-8-1-feature-freeze/) 
- [x] Build (beta) https://drive.google.com/file/d/1n5mWRbpt8tRV_9-4XcEi_vem1BfXFkHk/view?usp=share_link
- [x] Stabilization 
#### Remaining stabilization tasks
- [x] https://github.com/PrestaShop/PrestaShop/issues/31851
- [x] #31979
- [x] #31833
~~Release GDPR Module (https://github.com/PrestaShop/psgdpr)~~
- [x] Product UI issues : https://github.com/PrestaShop/PrestaShop/pull/31968
  - [x] https://github.com/PrestaShop/PrestaShop/issues/31588
  - [x] https://github.com/PrestaShop/PrestaShop/issues/31589
- [x] https://github.com/PrestaShop/PrestaShop/pull/32379 
- [x] #32460
- [x] Push new wording towards crowdin

#### Release Candidate and final release
- [x] Build (RC1)
- [x] Release Candidate 1
  - [x] Backport a fix into 8.1 release candidate branch : [#32936](https://github.com/PrestaShop/PrestaShop/issues/32936)
- [x] QA and community validation
- [x] Final release

#### Available for upgrade
- [x] Finalize autoupgrade --> https://github.com/PrestaShop/autoupgrade/milestone/26

### PR DESCRIPTION
**Pull request by lartist**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.1.x
| Description?      | Fix wordings
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | auto tests 🟢 https://github.com/lartist/ga.tests.ui.pr/actions/runs/4566106306
| Fixed ticket?     | Fixes #30719
| Related PRs       | https://github.com/PrestaShop/PrestaShop/pull/31935
| Sponsor company   | Your company or customer's name goes here (if applicable).

### CODE REVIEW

File: mails/themes/modern/core/account.html.twig [hunk 1]
```diff
@@ -248,7 +248,7 @@
                             <tbody>
                               <tr>
                                 <td align="center" style="font-size:0px;padding:10px 25px;word-break:break-word;">
-                                  <div style="font-family:Open sans, arial, sans-serif;font-size:14px;line-height:25px;text-align:center;color:#363A41;">{{ 'You can now place orders on our shop:'|trans({}, 'Emails.Body', locale)|raw }} <a href="{shop_url}" target="_blank" style="text-decoration:none;color:#25B9D7; display:block">{shop_name}</a></div>
+                                  <div style="font-family:Open sans, arial, sans-serif;font-size:14px;line-height:25px;text-align:center;color:#363A41;">{{ 'You can now place orders on our store:'|trans({}, 'Emails.Body', locale)|raw }} <a href="{shop_url}" target="_blank" style="text-decoration:none;color:#25B9D7; display:block">{shop_name}</a></div>
```
Comment (jolelievre) [inline mails/themes/modern/core/account.html.twig:251]: Hi @lartist,

sorry but this is a special case (again 😅) the templates for the modern theme are actually generated, they are based on MJML templates, a markup language for email templates which is then compiled. This is not blocking for the PR don't worry, the wordings can be updated manually as a temporary solution.

However, if we don't want these wordings to be overridden the initial sources must be updated as well, and exported in the core in a second PR to make sure they are fully synced.

Comment (jolelievre) [reply mails/themes/modern/core/account.html.twig:251]: You can find a bit of details in this mail theme README https://github.com/PrestaShop/PrestaShop/blob/develop/mails/themes/modern/README.md

This is the tool that compiles MJML templates into twig ones https://github.com/PrestaShop/mjml-theme-converter You will have to update the content of the source theme which is in the same repo as the tool https://github.com/PrestaShop/mjml-theme-converter/tree/master/mails/themes/modern_mjml A PR on the repo will be needed and in parallel, you'll have to run the command (detailed in the README) to update the modern theme in the core repository (and make another PR)

Comment (jolelievre) [reply mails/themes/modern/core/account.html.twig:251]: I hope the explanation are clear enough, don't hesitate to ask for more details if needed

**Review by jolelievre — APPROVED** [review]
Thanks @lartist 

All good however I detailed another task related to the modifications on the modern mail theme that will have to be done in two other PRs

File: mails/themes/modern/core/account.html.twig (hunk 1 above)
Comment (lartist) [reply mails/themes/modern/core/account.html.twig:251]: ok

**Comment by l-delin** [comment]
Hello, the wording looks good to me :-) thank you

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
@PrestaShop/qa-team please confirm that it can be merged without tests

**Comment by HanaRebaiQA** [comment]
Hello @kpodemski 

I saw that the auto tests are green and the CI are well passed
No need to test it

**Comment by kpodemski** [comment]
thanks @lartist

**Pull request merged**